consensus/reactor_test.go

```diff
@@ -140,11 +146,21 @@ func TestVotingPowerChange(t *testing.T) {
 }
 
 func TestValidatorSetChanges(t *testing.T) {
+	// FIXME: need to block css[j] until we check all txs! otherwise validator
```

Comment:
Though it would still be a hack, we might be able to use TimeoutCommit here instead of the new param/sleep. TimeoutCommit  is the time a node waits after committing a block, before it goes into the next height. So it will finish everything from the last block, but then wait a bit. The idea is this gives it time to hear more votes from other validators, to strengthen the commit it includes in the next block. But it also gives it time to hear about new transactions.

Regardless, its not good that any of our tests depend on time, and we'll need to design something else.